packages/server/test/functional/utils/index.ts

```diff
@@ -1,10 +1,23 @@
+import path from 'path'
 import { v4 as uuidv4 } from 'uuid'
 import { App } from '../../../src/app'
 
 let app: App
 
 const setupApp = async () => {
-  process.env.DATABASE_URL = `.test-data/${uuidv4()}.sqlite`
+  process.env.SKIP_LOAD_ENV = 'true'
+  process.env.SUPPRESS_LOGGING = 'true'
+  process.env.DATABASE_URL = path.join(
+    __dirname,
+    '..',
+    '..',
+    '..',
+    '..',
+    '..',
+    'test',
+    '.test-data',
+    `${uuidv4()}.sqlite`
+  )
```

Comment:
SKIP_LOAD_ENV is important so tests aren't affected by our current config. I added a SUPPRESS_LOGGING option so that we stop seeing `Table created` messages all the time.